src/components/Header.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import SearchBar from './searchBar';
+
+interface HeaderProps {
```

Comment:
Since we have single property, can we call it as `HeaderProp` ?
```suggestion
interface HeaderProp {
```

Reply:
fixed https://github.com/Sakshi-Simform/Recipe-App/pull/3/commits/4ca42c983324fb80aeaf93004af1110079fcd65e